lib/tasks/cleanup_vulnerabilities.rake

```diff
@@ -0,0 +1,37 @@
+desc 'Cleanup old unused vulnerabilities data'
+task cleanup_vulnerabilities: :environment do
+  @conn = ActiveRecord::Base.connection
+  Project.where.not(best_project_security_set_id: nil).select(:id, :best_project_security_set_id).find_each do |project|
+    begin
+      pss_ids = ProjectSecuritySet.where(project_id: project.id)
+                                  .where('id < ?', project.best_project_security_set_id).select(:id).pluck(:id)
```

Comment:
Isn't thing potentially dangerous when DB IDs wrap around?  We would have old ID's that are larger than the `project.best_project_security_set_id` that are *supposed* to be removed, but won't be with this logic. Is there a `created_at` field we can use?